src/pydata_sphinx_theme/__init__.py

```diff
@@ -1084,6 +1085,43 @@ def setup_translators(app):
             app.set_translator(name, translator, override=True)
 
 
+# ------------------------------------------------------------------------------
+# customize method for logo management
+# ------------------------------------------------------------------------------
+
+
+def setup_logo_path(
+    app: Sphinx, pagename: str, templatename: str, context: dict, doctree: nodes.Node
+) -> None:
+    """Set up relative paths to logos (works for all Sphinx versions)."""
+
+    # get informations from the context
+    pathto = context.get("pathto")
+    logo = context.get("logo_url", "") or context.get("logo", "")
+    image_light = context.get("theme_logo", {}).get("image_light")
+    image_dark = context.get("theme_logo", {}).get("image_dark")
+
+    # add the theme_logo to context to make it editable downstream
+    if "theme_logo" not in context:
+        context["theme_logo"] = {}
+
+    # create the logo_url
+    if logo and not isurl(logo):
+        logo = pathto(f"_static/{logo}", resource=True)
+
+    # update light image with the appropriate url
+    if image_light and not isurl(image_light):
+        image_light = pathto(f"_static/{image_light}", resource=True)
+    image_light = image_light or logo
+    context["theme_logo"]["logo_light"] = image_light
```

Comment:
```suggestion
    context["theme_logo"]["image_light"] = image_light
```

Reply:
this very one is a feature. If you set `image_light`, you replace it in the context. And as i'm using it as a source at the begining of the script, every page will add "_static/../_static/..". You have no idea about the links I got when I was testing it.